Softeq.XToolkit.Permissions.iOS/Softeq.XToolkit.Permissions.iOS.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0-ios10.0</TargetFramework>
+    <TargetFramework>net6.0-ios14.0</TargetFramework>
```

Comment:
Looks like ios13 will be enough,
only bluetooth requires ios13+

Reply:
changed